.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-   - "0.10"
-
+  - "0.10"
```

Comment:
Just cleaning up formatting here. One change to consider though (if you haven't already) is to add "0.11" as a node version to test, but to allow failures (since it is an unstable release. This will let you monitor potentially breaking changes. See the [express](https://github.com/strongloop/express/blob/master/.travis.yml) configuration.